src/SortingTool.java

```diff
@@ -0,0 +1,16 @@
+import engine.Config;
+import errors.CommandException;
+
+public class SortingTool {
```

Comment:
Currently there is SortingTool and Tool. And it is not clear which of this class contain`main` function
Please, rename SortingTool to `SortingToolRunner` vs `SortingToolMain`